### PR TITLE
Fix issues in OpenShift support

### PR DESCRIPTION
**Pull request by lhotari**
This PR contains changes on top of PR #12 to support OpenShift.

- support arbitrary user id that OpenShift uses by default. Achieved by these lines in Dockerfile
```
RUN chown -R user:0 /home/user && chmod g=u /home/user
ENV HOME=/home/user
```
- fix issue in loading configuration file. Use `ENV PULSAR_OPS_MONITOR_CFG=/config/runtime.yml` to specify config file.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -19,15 +35,25 @@ RUN go get github.com/google/gops
...
 # a kesque cert but it can overwritten by mounting the same path ca-bundle.crt
 COPY --from=builder /root/config/kesque-pulsar.cert /etc/ssl/certs/ca-bundle.crt
 
 # Copy debug tools
-COPY --from=builder /go/bin/gops /app/gops
-
-ENTRYPOINT ./pulsar-heartbeat ./runtime.yml
+COPY --from=builder /go/bin/gops /home/user/gops
+RUN mkdir /home/user/run
+RUN chown -R user:0 /home/user && chmod g=u /home/user
```
Comment (michaeljmarshall) [inline Dockerfile:52]: @lhotari - this line adds 62 mb to the docker image. I know that OpenShift requires specific file permissions, but does the process actually write to all of the directories? If it does, we can use the chown argument to the copy command: `COPY --chown=user:0`. If it doesn't, then we can chown just the few directories/files that the process needs to write to.

For reference, I use the `dive` tool to quickly inspect docker images. Here is an output for `dive datastax/pulsar-heartbeat:1.0.4`:

Screen Shot 2021-07-06 at 9 04 23 AM

**Review by michaeljmarshall — COMMENTED** [review]
I know we already merged this. Just leaving a comment for a way to make the docker image smaller.

File: Dockerfile (hunk 1 above)
Comment (lhotari) [reply Dockerfile:52]: I created a follow-up PR #15 to address that separately.

Comment (lhotari) [reply Dockerfile:52]: you can check with `dive lhotari/pulsar-heartbeat:rootlessv3` , that's the built image off PR #15 . The total image size is now 66MB.

Comment (lhotari) [reply Dockerfile:52]: thanks for introducing [dive](https://github.com/wagoodman/dive), very useful tool.

Comment (michaeljmarshall) [reply Dockerfile:52]: @lhotari - https://github.com/datastax/pulsar-heartbeat/pull/15 looks good to me. 

> thanks for introducing dive, very useful tool.

Glad you found it useful. I really like it when trying to optimize a docker image's size.

**Pull request merged**